app/views/assignments/_boot.js.erb

```diff
@@ -45,7 +45,7 @@
     $('.assignment_file').each(function() {
       if (!filename_re.exec(this.value)) {
         invalid = true;
-        $(this).css('background-color', var(--heavy_error));
+        $(this).css('background-color', severe-error);
```

Comment:
Good catch, though this fix is a syntax error (`severe-error` isn't a JS variable).

We want to grab the colour off of the set style for the theme, something like:

```javascript
$(this).css('background-color', document.documentElement.style.getPropertyValue('--severe_error'));


```